.github/workflows/ci.yaml

```diff
@@ -53,8 +53,8 @@ jobs:
 
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip
           pip install six
+          python -m pip install --upgrade pip
```

Comment:
Comment is needed for this workaround. Also this will start to fail if the default pip is updated on the host image? Shouldn't we bite the bullet?